Makefile

```diff
@@ -409,6 +404,19 @@ flush:
 php-shell:
 	@docker-compose exec php-fpm bash
 
+.PHONY: installnpm
+installnpm:
+  # Update packages
+	docker-compose exec php-fpm apt update
+	# Install NPM
+	docker-compose exec php-fpm apt install npm -y
+	# Update Node version
+	docker-compose exec php-fpm sh -c 'npm cache clean -f && npm install -g n && n stable'
```

Comment:
Is the plan to add `installnpm` to the `dev` target? In that case we need to adjust the [repos.sh](repos script) to use the npm from the container for each repo, right?